project.clj

```diff
@@ -9,14 +9,14 @@
 
   :dependencies [[clj-time "0.11.0"]
                  [functionalbytes/clj-ip "0.9.0"]
-                 [org.clojure/clojure "1.7.0"]
-                 [org.clojure/core.cache "0.6.4"]
+                 [org.clojure/clojure "1.10.1"]
```

Comment:
I am afraid to change clojure version. as you know when clojure 1.9 released, there is some change on namespace `spec` . So, if you don't need `1.10.1`  I sugges keeping `1.7`

Reply:
Fair point, changing clojure version is definitely scary. But this is a small library, the tests run fine, and it's going to end up using clojure 1.10.1 anyway when we include it in our main project, so I would rather keep it on the latest clojure to avoid further pain.